### ISSUE TITLE
MAKE THE PROJECT PRIVATE NEBA

### ISSUE THREAD

**Issue by Azizadx** [nebiyuelias1/phison-realestate#136]
the project isnot private neba

**Comment by nebiyuelias1** [comment]
Yes, I did that intentionally. The good thing is that I'm not exposing any API keys or secrets. People can take a look at it but they won't be able to do any harm.

**Comment by Azizadx** [comment]
alright cool
